rest/src/main/java/discord4j/rest/util/MultipartRequest.java

```diff
@@ -27,28 +27,35 @@
 import java.util.List;
 
 public class MultipartRequest {
-
-    private final MessageCreateRequest createRequest;
+    @Nullable
+    private final Object jsonPayload;
```

Comment:
Was considering making this generic. Thoughts?

Reply:
Would there be anything to gain from giving WebhookExecuteRequest and MessageCreateRequest a closer least upper bound? Would be a discord-json change.

Reply:
I was also thinking of something like that. Maybe something like BaseMessageCreateRequest?